pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
In this case I'm going to have to object to halting the bot -- the idea is that a plugin failure should have no impact on the main bot process. Perhaps there is a way to make failures more obvious?